python/otel-metrics-collector.Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Use lightweight Python image
+FROM python:3.9-slim
+
+# Install dependencies
+RUN pip install requests opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp prometheus_client
+
+# Copy the script into the container
+WORKDIR /app
+COPY otel_metrics_collector.py .
+
+# Run the script
+CMD ["python", "otel_metrics_collector.py"]
```

Comment:
not sure if we need this, Otel collector has a way to inject the sidecar to collect metrics
https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#sidecar-injection